docs/Training-PPO.md

```diff
@@ -166,6 +166,16 @@ variables, this should be larger.
 
 Typical Range: `32` - `512`
 
+### (Optional) Visual Encoder Type
+
+`vis_encode_type` correspond to the encoder type for encoding visual observations.
+Valid options include:
+* `default` (default): a simple encoder consists of two CNN layer
+* `nature_cnn`: Mnih's CNN implementation (https://www.nature.com/articles/nature14236)
```

Comment:
nit: Mnih et al. for the author name. Also probably good to have the # of layers for Nature and ResNet so people have a general idea of which one to choose without reading the papers.